docroot/profiles/custom/cgov_site/modules/custom/pdq_cancer_information_summary/pdq_cancer_information_summary.tokens.inc

```diff
@@ -34,6 +34,7 @@ function pdq_cancer_information_summary_tokens($type, $tokens, array $data, arra
         case 'cgdp-field-pdq-is-svpc':
           $replacements[$original] = get_svpc_summary($data);
           break;
+
```

Comment:
Please don't change this file in this commit -- it has nothing to do with the domain token.